src/shared/conf-parser.c

```diff
@@ -2002,6 +2002,44 @@ int config_parse_in_addr_data(
         return 1;
 }
 
+
+int config_parse_in_addr_prefix(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype, /* takes boolean, whether we warn about missing prefixlen */
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        struct in_addr_prefix *p = ASSERT_PTR(data);
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+
+        if (isempty(rvalue)) {
+                *p = (struct in_addr_prefix) {};
+                return 1;
+        }
+
+        r = in_addr_prefix_from_string_auto_internal(rvalue, ltype ? PREFIXLEN_REFUSE : PREFIXLEN_FULL, &p->family, &p->address, &p->prefixlen);
```

Comment:
hmm if this function is called now from other code, it should probably be called `xyz_full()` not `xyz_internal()` since it apparently is not internal